bot/cogs/monstersurvey.py

```diff
@@ -0,0 +1,161 @@
+from discord.ext.commands import Bot, Context
+from asyncio import sleep as asleep
+from typing import Optional
+from discord.ext import commands
+from discord import Embed
+import logging
+import json
+import os
+
+log = logging.getLogger(__name__)
+
+EMOJIS = {
+    'SUCCESS': u'\u2705',
+    'ERROR': u'\u274C'
+}
+
+
+class MonsterSurvey:
+    """
+    Vote for your favorite monster!
+    This command allows users to vote for their favorite listed monster.
+    Users may change their vote, but only their current vote will be counted.
+    """
+
+    def __init__(self, bot: Bot):
+        """Initializes values for the bot to use within the voting commands."""
+        self.bot = bot
+        self.registry_location = os.path.join(os.getcwd(), 'resources', 'monstersurvey', 'monstersurvey.json')
```

Comment:
Needs to use the correct path:

`os.path.join(os.getcwd(), 'bot', 'resources', 'monstersurvey', 'monstersurvey.json')`

Reply:
For clarification, do you want a path such as ../resources/monstersurvey/monstersurvey.json or are your equesting that monstersurvey.json live in the resources directly instead of its own subfolder?

Reply:
Neither. The bot is invoked as a python module, `os.getcwd()` is going to return the root of this repository. `./resources` doesn't exist, `./bot/resources` does.